### ISSUE TITLE
Flaky test Submit Add Family form

### ISSUE THREAD

**Issue by dianabarsan** [medic/cht-core#8236]
The test has been failing repeatedly with 

```
1) Family form Submit Add Family form
[chrome 113.0.5672.63 linux #0-15] element ("input[name="/data/clinic/household_survey/source_of_drinking_water"]") still not clickable after 15000ms
[chrome 113.0.5672.63 linux #0-15] Error: element ("input[name="/data/clinic/household_survey/source_of_drinking_water"]") still not clickable after 15000ms
```

Repeat failures are: 
```
[chrome 113.0.5672.63 linux #0-15] 2) Family form Submit Add Family form
[chrome 113.0.5672.63 linux #0-15] element (".fast-action-trigger .fast-action-flat-button") still not clickable after 15000ms
[chrome 113.0.5672.63 linux #0-15] Error: element (".fast-action-trigger .fast-action-flat-button") still not clickable after 15000ms
[chrome 113.0.5672.63 linux #0-15]     at async clickFastActionFlat (/home/runner/work/cht-core/cht-core/tests/page-objects/default/common/common.wdio.page.js:78:3)
[chrome 113.0.5672.63 linux #0-15]     at async Object.openFastActionReport (/home/runner/work/cht-core/cht-core/tests/page-objects/default/common/common.wdio.page.js:91:5)
[chrome 113.0.5672.63 linux #0-15]     at async Context.<anonymous> (/home/runner/work/cht-core/cht-core/tests/e2e/default/enketo/add-family.wdio-spec.js:36:5)
```

https://github.com/medic/cht-core/actions/runs/4951831780/jobs/8858884875#step:16:648

**Comment by dianabarsan** [comment]
I've disabled the test in `master`.

**Comment by tatilepizs** [comment]
Fixed on PR #8720
